### PR TITLE
CI Updates

### PR DESCRIPTION
**Pull request by hsachdevah**
1. Default infrastructure tfvars to platform.tfvars with required overrides
2. Set create_network = true
3. Remove T4 GPU pool from platform.tfvars

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -287,10 +289,15 @@ steps:
...
-        terraform destroy -var-file=tfvars_tests/standard-gke-public.platform.tfvars -var=project_id=$PROJECT_ID \
+        terraform destroy \
+        -var-file=platform.tfvars \
+        -var=project_id=$PROJECT_ID \
+        -var=network_name=ml-network-$_BUILD_ID \
+        -var=subnetwork_name=ml-network-$_BUILD_ID \
         -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
         -var=autopilot_cluster=$_AUTOPILOT_CLUSTER \
-        -var=cluster_location=$_REGION -auto-approve -no-color
+        -var=cluster_location=$_REGION \
```
Comment (andrewsykim) [inline cloudbuild.yaml:299]: Also consider changing the region to us-east4

File: infrastructure/platform.tfvars [hunk 2]
```diff
@@ -43,30 +43,17 @@ cpu_pools = [{
...
   max_count          = 3
   disk_size_gb       = 100
-  disk_type          = "pd-standard"
+  disk_type          = "pd-balanced"
+  enable_gcfs        = true
+  logging_variant    = "DEFAULT"
   accelerator_count  = 2
-  accelerator_type   = "nvidia-tesla-t4"
+  accelerator_type   = "nvidia-l4"
   gpu_driver_version = "DEFAULT"
```
Comment (andrewsykim) [inline infrastructure/platform.tfvars:58]: This was LATEST in `infrastructure/tfvars_tests/standard-gke-public.platform.tfvars`, does the value matter?

Comment (hsachdevah) [reply infrastructure/platform.tfvars:58]: This is not an issue.

File: cloudbuild.yaml (hunk 1 above)
Comment (hsachdevah) [reply cloudbuild.yaml:305]: Done

**Comment by spencer-p** [comment]
Closing old issues with merge conflicts. Please rebase and re-open if still relevant.

**Pull request closed without merging**